debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/j9/J9ConfigFlags.java

```diff
@@ -43,6 +43,7 @@
 	public static final boolean arch_power;
 	public static final boolean arch_s390;
 	public static final boolean arch_x86;
+	public static final boolean arch_riscv;
```

Comment:
Please keep the declarations here (and the initializations below) in sorted order.

Reply:
Agreed and updated as suggested above.

In addition,  I am still writing code to enable the basic functionalities/commands of DDR and will add all my changes to this PR later once they are confirmed to work good on RISC-V as expected.